daprdocs/content/en/getting-started/get-started-api.md

```diff
@@ -6,33 +6,31 @@ weight: 30
 description: "Run a Dapr sidecar and try out the state API"
 ---
 
-Running [`dapr init`]({{<ref install-dapr-selfhost.md>}}) loads your local environment with:
+In this guide, you'll simulate an application by running the sidecar and calling the API directly. After running Dapr using the Dapr CLI, you'll:
 
-- The Dapr sidecar binaries.
-- Default Redis component definitions for both:
-  - State management, and
-  - A message broker.
+- Store two state object names.
+- Perform a bulk get on the two names.
+- Run a transaction operation, adding a third name and removing the first name.
+- Delete the state object names.
```

Comment:
Need to clean up this section to say that it is save, read and then delete and only for ONE object. Below is says objects and states (plural)